joplin/pages/topic_page/models.py

```diff
@@ -26,7 +26,7 @@ class TopicPage(JanisBasePageWithTopicCollections):
     base_form_class = TopicPageForm
 
     publish_requirements = (
-        RelationPublishRequirement('top_pages'),
+        # RelationPublishRequirement('top_pages'),
```

Comment:
are we removing this requirement?

Reply:
Yes! I'd noticed this before. But making top_pages a requirement creates a circular dependency. It hasn't really affected production yet, because a bunch of topics and topic collections were grandfathered in before we implemented publish_requirements.

We can't publish a page without a live topic.
We can't have publish a topic with a live page.

So there isn't a way to create a new topic and a new page for a new topic. Unless I'm crazy and missing something. That was something I wanted to bring up with the team.

Reply:
you arent crazy, for me to add pages to an empty database, id have to mark the pages as "global" so I could publish them and be live and then add them to topic.  Now I could publish the topic. Then go back to page and uncheck top page/global and make the topic be its parent. snake eating its tail indeed.

Reply:
I would do the same as @chiaberry when building out test data